app/static/css/main.css

```diff
@@ -222,60 +236,73 @@ input {
     text-align: left;
 }
 
-.inputOuterContainer {
+.input_outer_container {
     width: 30rem;
     margin-bottom: 2rem;
     padding-top: 6rem;
 }
 
-.itemInput {
+.item_input {
     border-style: solid;
     border-color: rebeccapurple;
     border-radius: 5px;
     padding: 1rem;
     margin-left: 1rem;
 }
 
-.itemTextInput {
+.item_text_input {
     border-radius: 5px;
     width: 98%;
     height: 2rem;
     font-size: 1rem;
 }
 
-.itemText {
+.item_text {
     padding-top: 0.3rem;
     color: rebeccapurple;
     font-weight: bold;
     display: flex;
 }
 
-.itemText {
+.item_text {
     text-transform: capitalize;
 }
 
-.inputContainer div {
+.input_container div {
     padding-bottom: 0.5rem;
     font-weight: bold;
     font-size: large;
     text-decoration-line: underline;
     color: rebeccapurple;
 }
 
-.inputContainer textarea {
+.input_container textarea {
     width: 98%;
     height: 3rem;
 }
 
 .item_box {
-    height: auto;
-    padding-right: 1rem;
-    border-style: solid;
-    border-width: 7px;
-    border-color: rebeccapurple;
-    border-radius: 5px;
-    padding: 0.5rem;
-    margin-bottom: 1rem;
+    width: 40rem;
+    padding: 1rem;
+    border-radius: 7px;
+    border: rebeccapurple solid 2px;
+    /* background: #EC985A; */
+    color: white;
```

Comment:
`color` changes the font color correct?

Reply:
Ah not sure what happened there, sure I went through and deleted all comented out code but must have either fogotten to commit or forgot to save. Changed now.